index.html

```diff
@@ -0,0 +1,380 @@
+                            <a class="card__icon" href="#"><img src="/vectors/Vector (14).svg" alt=""></a>
+
+                        </div>
+                    </div>
+                </div>
+
+                <div class="card">
+                    <img class="card__img" src="/img/cover (2).png" alt="">
+                    <div class="card__body">
+                        <h5 class="text-card">Avie Beaton</h5>
+                        <p class="professional professional--color">CO Founder</p>
+
+                        <div class="card__footer">
+                            <a class="card__icon" href="#"><img src="/vectors/Vector (12).svg" alt=""></a>
+                            <a class="card__icon" href="#"><img src="/vectors/Vector (13).svg" alt=""></a>
+                            <a class="card__icon" href="#"><img src="/vectors/Vector (14).svg" alt=""></a>
+                        </div>
+                    </div>
+                </div>
+        </section>
+
+        <section class="main-page__form-button">
+
+            <div class="main-page__form">
+
+                <div class="main-page__contact">
+                    <p class="main-page__contact-text">Contact Us</p>
+                    <h2 class="main-page__contact-header">Make an Appointment</h2>
+                </div>
+
+                <form class="form">
+
+                    <div class="form__group">
+                            <input type="text" class="form__input" placeholder="Full Name">
+                            <input type="text" class="form__input" placeholder="example@gmail.com">
+                    </div>
+
+                    <div class="form__group">
+                            <input type="text" class="form__input" placeholder="Please Select">
+                            <input type="text" class="form__input" placeholder="4:00 Available">
+                    </div>
+
+                    <div class="form__group">
+                        <textarea class="form__text"
+                            placeholder="Message..."></textarea>
+                    </div>
+                    <button class="btn btn--rectangle">Book Appointment</button>
+                </form>
+            </div>
+
+            <div class="main-page__background main-page-background--color">
+                <div class="main-page__consulting">
+                    <div class="main-page__consulting-content">
+                        <h2>Consulting Agency For Your Business</h2>
+                        <p class="text text--alignment">the quick fox jumps over the lazy dog</p>
+                    </div>
+                    <div>
+                        <button class="btn btn--rectangle">Contact Us</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+    </main>
+
+
+    <footer class="footer">
+        <div class="footer-header">
+            <h1 class="header__logo header__logo--color-white">KMarketer</h1>
```

Comment:
`header__logo `is positioned outside the Block (header).
`header__logo--color-white` is positioned outside the Block (header).